### PR TITLE
Review info page

### PR DESCRIPTION
**Pull request by enyia21**

### CODE REVIEW

File: src/main/resources/messages.properties [hunk 1]
```diff
@@ -407,6 +407,14 @@ next-steps-email.body=<html><body><p><b>Time to Hear Back:</b><br>In the next 7-
 #
 phone-number-nudge.title=Phone number nudge
 phone-number-nudge.header=Are you sure you want to leave your phone number blank?
-phone-number-nudge.subheader=A caseworker may need to contact you by phone about your application. If you don't have a phone number, you can enter a friend or family member's phone number instead.
+phone-number-nudge.subheader=A caseworker may need to contact you by phone about your application. If you don't have a phone number, you can enter a friend or family member's phone number instead.
```
Comment (coltborg) [inline src/main/resources/messages.properties:410]: Small nick pick: remove the white space at the end of the line.

```suggestion
phone-number-nudge.subheader=A caseworker may need to contact you by phone about your application. If you don't have a phone number, you can enter a friend or family member's phone number instead.
```

File: src/main/resources/templates/ubi/phoneNumberNudge.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
       <div th:replace="~{fragments/goBack :: goBackLink}"></div>
       <main id="content" role="main" class="form-card spacing-above-35">
         <div class="form-card__content">
-          <th:block th:replace="~{'fragments/icons' :: docWithMagnifyingGlass}"></th:block>
+          <th:block th:replace="~{'fragments/icons' :: contactInfo}"></th:block>
```
Comment (coltborg) [inline src/main/resources/templates/ubi/phoneNumberNudge.html:12]: @enyia21 did you get feedback to change the icon? From the Figma, I expected it to be `docWithMagnifyingGlass`. Maybe I missed a comment somewhere.

**Review by coltborg — CHANGES_REQUESTED** [review]
Make a few comments.

File: src/main/resources/templates/ubi/phoneNumberNudge.html (hunk 2 above)
Comment (enyia21) [reply src/main/resources/templates/ubi/phoneNumberNudge.html:12]: You are correct about the change in the icons.   I'm pushing code with the correct Icon.

File: src/main/resources/messages.properties (hunk 1 above)
Comment (enyia21) [reply src/main/resources/messages.properties:410]: @coltborg I'm pushing code to fix this.

**Review by coltborg — APPROVED** [review]
Comments addressed, approved.

**Pull request merged**